source/common/runtime/runtime_impl.cc

```diff
@@ -23,60 +23,78 @@ namespace Runtime {
 
 const size_t RandomGeneratorImpl::UUID_LENGTH = 36;
 
+void RandomGeneratorImpl::random_bytes(uint8_t* out, size_t out_len) {
```

Comment:
I was actually referring to edge cases on the allocations, not the randomness. For example allocating > 1024 bytes (specifically 2048) which I don't think is covered right now? Also, allocating 2047, then 16, etc.